docs/bit-csharp-client/check-box.md

```diff
@@ -0,0 +1,51 @@
+# Bit Check Box
+
+**BitCheckBox** is a fully customizable control .It supports all Xamarin Forms backends, including but not limited to Android-iOS-Windows(UWP) etc.
+
+### Getting Started
+
+In order to use BitCheckBox, install "Bit.CSharpClient.Controls" nuget package.
```

Comment:
Please add a new file which describes how to install nuget package & required call to BitCSharpClientControls.Init. Instead of providing this info in every control!
It can also contains xaml namespace step too (-: